src/transformers/utils/hub.py

```diff
@@ -1081,6 +1093,27 @@ def extract_info_from_url(url):
     return {"repo": cache_repo, "revision": revision, "filename": filename}
 
 
+def create_and_tag_model_card(repo_id, tags=None):
+    """
+    Creates a dummy model card and tags it.
+    """
+    model_card = None
+
+    try:
+        # Check if the model card is present on the remote repo
+        model_card = ModelCard.load(repo_id)
+    except EntryNotFoundError:
+        # Otherwise create a simple model card from template
+        card_data = ModelCardData(language="en", tags=[])
+        model_card = ModelCard.from_template(card_data)
```

Comment:
would be nice to set a default `model_summary` field here with some info along the lines "this is a *** model. It has been pushed to the Hub using `transformers`. This model card has been auto-generated."

Reply:
Makes sense, done !